Source/Pawnmorphs/Esoteria/Utilities/Collections/ExposedList.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Verse;
+
+namespace Pawnmorph.Utilities.Collections
+{
+	internal class ExposedListContainer<T> : IExposable, IEnumerable<T>, IList<T>
```

Comment:
What was the reason for creating this class?  I thought Scribe could handle regular lists already

Reply:
It does, but only if you scribe it directly. However it doesn't seem to be able to handle nested collections.
In this case a dictionary of lists.
HAR added a scribe nested collections, but It only seem to handle nested hashsets.

Reply:
Ah, that makes sense!